testsuite/tests/conftest.py

```diff
@@ -811,7 +824,7 @@ def __init__(self):
         def _custom_service(
             self,
             params,
-            proxy_params=None,
+            proxy_params={},
```

Comment:
This is wrong, you should not have a mutable default value.

Reply:
You're faster than pylint!!!